src/handler/worksheet.ts

```diff
@@ -4,11 +4,29 @@ import { handlerCell, relevantStyle } from './cell.ts';
 import { Document, Element } from '@borgar/simple-xml';
 import { ConversionContext } from '../ConversionContext.ts';
 import type { Rel } from './rels.ts';
-import type { Worksheet } from '@jsfkit/types';
+import type { Worksheet, WorksheetLayoutScales, WorksheetView } from '@jsfkit/types';
 import { colWidth } from '../utils/colWidth.ts';
 import { fromA1 } from '../utils/fromA1.ts';
 import { toA1 } from '../utils/toA1.ts';
 import { getFirstChild } from '../utils/getFirstChild.ts';
+import { toInt } from '../utils/typecast.ts';
+
+/**
+ * Extracts zoom levels (layout scales) for the different view modes for a sheet.
+ *
+ * Excel stores separate zoom percentages for normal view, page layout view, and page break preview.
+ * Only non-default zoom values are included in the returned object.
+ */
+function getLayoutScales (sheetView: Element): WorksheetLayoutScales {
+  const normalScale = toInt(attr(sheetView, 'zoomScaleNormal'));
+  const pageLayoutScale = toInt(attr(sheetView, 'zoomScalePageLayoutView'));
+  const pageBreakPreviewScale = toInt(attr(sheetView, 'zoomScaleSheetLayoutView'));
+  return {
+    ...(normalScale && { normal: normalScale }),
+    ...(pageLayoutScale && { pageLayout: pageLayoutScale }),
+    ...(pageBreakPreviewScale && { pageBreakPreview: pageBreakPreviewScale }),
+  };
```

Comment:
I find this spread composition method hard to read. Worse, however, is how slow/inefficient it is. So I would much prefer composing object old-school style:

```suggestion
  const scales: WorksheetLayoutScales = {};
  const normalScale = toInt(attr(sheetView, 'zoomScaleNormal'));
  if (normalScale != null) { scales.normal = normalScale; }
  const pageLayoutScale = toInt(attr(sheetView, 'zoomScalePageLayoutView'));
  if (pageLayoutScale != null) { scales.pageLayout = pageLayoutScale; }
  const pageBreakPreviewScale = toInt(attr(sheetView, 'zoomScaleSheetLayoutView'));
  if (pageBreakPreviewScale != null) { scales.pageBreakPreview = pageBreakPreviewScale; }
  return scales;
```